contracts/strategies/direct-grants-simple/DirectGrantsSimpleStrategy.sol

```diff
@@ -352,7 +333,7 @@ contract DirectGrantsSimpleStrategy is BaseStrategy, ReentrancyGuard {
             IAllo.Pool memory pool = allo.getPool(poolId);
             allocatedGrantAmount += grantAmount;
 
-            if (allocatedGrantAmount > pool.amount) {
+            if (allocatedGrantAmount > poolAmount) {
```

Comment:
Okie this makes sense cause poolAmount is updated anytime funding is increased right ? It might be nicer to read the amount always from Allo.sol but that does cost more gas and this instance -> doesn't make sense